internal/Test-PSRemoting.ps1

```diff
@@ -5,34 +5,34 @@ Function Test-PSRemoting {
 Jeff Hicks
 https://www.petri.com/test-network-connectivity-powershell-test-connection-cmdlet
 #>
-  [cmdletbinding()]
- 
-  Param(
-    [Parameter(Position=0,Mandatory,HelpMessage = "Enter a computername",ValueFromPipeline)]
-    [ValidateNotNullorEmpty()]
-    [string]$Computername,
-    $Credential = [System.Management.Automation.PSCredential]::Empty
-  )
- 
-  Begin {
-    Write-Message -Level Verbose -Message "Starting $($MyInvocation.Mycommand)"
-  } #begin
- 
-  Process {
-    Write-Message -Level Verbose -Message "Testing $computername"
-    Try {
-      $r = Test-WSMan -ComputerName $Computername -Credential $Credential -Authentication Default -ErrorAction Stop
-      $True 
-    }
-    Catch {
-      Stop-Function -Message "Remote testing failed for computer $ComputerName" -Target $ComputerName -ErrorRecord $_ -Continue
-      return $false
-    }
-    
-  } #Process
- 
-  End {
-    Write-Message -Level Verbose -Message "Ending $($MyInvocation.Mycommand)"
-  } #end
- 
+	[cmdletbinding()]
+	param(
+	[Parameter(Position=0,Mandatory,HelpMessage = "Enter a computername",ValueFromPipeline)]
+	[ValidateNotNullorEmpty()]
+	[string]$Computername,
+	$Credential = [System.Management.Automation.PSCredential]::Empty,
+	[switch]$Silent
+	)
+
+	begin {
+		Write-Message -Level VeryVerbose -Message "Starting $($MyInvocation.Mycommand)"
+	} #begin
+
+	process {
+		Write-Message -Level VeryVerbose -Message "Testing $computername"
+		try {
+			$r = Test-WSMan -ComputerName $Computername -Credential $Credential -Authentication Default -ErrorAction Stop
+			$True 
+		}
+		catch {
+			Stop-Function -Message "Remote testing failed for computer $ComputerName" -Target $ComputerName -ErrorRecord $_
+			return $false
```

Comment:
Consider whether this is how you want to use it. It's a significant improvement over `-Continue`, but do you want to throw on `-Silent`
It's a test command, so it may be appropriate to just return false.
In that case, you can use Write-Message to log the error record instead.
In order to find the right answer here, you'd have to check how this command is used by our other commands and whether they are designed to handle exceptions or boolean returns.